apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -101,4 +165,9 @@ if (!('BridgeServiceMixin' in self)) {
       { methods: METHODS, streams: STREAMS, events: EVENTS }
     )
   );
+
+  // Automatically init service if it's run inside worker.
+  if ('importScripts' in self) {
```

Comment:
BTW it seems not necessary to to check here since you already import the necessary scripts at first. Maybe we should just check if importScripts existed(or in worker scope) at the very beginning and return error message.